--- git status ---
On branch master
Your branch is up to date with 'origin/master'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   DX1_RL/DX1_RLPackage/DX1_RLSetup/tasks/velocity/velocity_env_cfg.py

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	DX1_RL/logs/rsl_rl/DX1_flat/2025-07-04_22-01-40/
	DX1_RL/logs/rsl_rl/DX1_flat/2025-07-04_22-08-33/
	DX1_RL/logs/rsl_rl/DX1_flat/2025-07-04_22-22-52/
	DX1_RL/outputs/

no changes added to commit (use "git add" and/or "git commit -a") 


--- git diff ---
diff --git a/DX1_RL/DX1_RLPackage/DX1_RLSetup/tasks/velocity/velocity_env_cfg.py b/DX1_RL/DX1_RLPackage/DX1_RLSetup/tasks/velocity/velocity_env_cfg.py
index 199d7cc..5c7c1e1 100644
--- a/DX1_RL/DX1_RLPackage/DX1_RLSetup/tasks/velocity/velocity_env_cfg.py
+++ b/DX1_RL/DX1_RLPackage/DX1_RLSetup/tasks/velocity/velocity_env_cfg.py
@@ -92,7 +92,7 @@ class CommandsCfg:
 
 @configclass
 class ActionsCfg:
-    joint_pos = mdp.JointPositionActionCfg(asset_name="robot", joint_names=[".*"], scale=0.7, use_default_offset=True)
+    joint_pos = mdp.JointPositionActionCfg(asset_name="robot", joint_names=[".*"], scale=0.4, use_default_offset=True)
 
 
 @configclass
@@ -231,8 +231,8 @@ class RewardsCfg:
     ang_vel_xy_l2 = RewTerm(func=mdp.ang_vel_xy_l2, weight=-0.05)
     dof_torques_l2 = RewTerm(func=mdp.joint_torques_l2, weight=-2.0e-5)
 
-    # dof_acc_l2 = RewTerm(func=mdp.joint_acc_l2, weight=-1.0e-7)
-    dof_acc_l2 = RewTerm(func=mdp.joint_acc_l2, weight=0.0)
+    dof_acc_l2 = RewTerm(func=mdp.joint_acc_l2, weight=-0.1e-7)
+    # dof_acc_l2 = RewTerm(func=mdp.joint_acc_l2, weight=0.0)
     
     action_rate_l2 = RewTerm(func=mdp.action_rate_l2, weight=-0.001)
     feet_air_time = RewTerm(