--- git status ---
On branch master
Your branch is up to date with 'origin/master'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   DX1_RL/DX1_RLPackage/DX1_RLSetup/tasks/velocity/velocity_env_cfg.py
	modified:   DX1_RL/logs/rsl_rl/DX1_flat/2025-07-04_20-58-32/exported/policy.pt

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	DX1_RL/logs/rsl_rl/DX1_flat/2025-07-04_21-30-52/
	DX1_RL/logs/rsl_rl/DX1_flat/2025-07-04_21-43-15/
	DX1_RL/outputs/

no changes added to commit (use "git add" and/or "git commit -a") 


--- git diff ---
diff --git a/DX1_RL/DX1_RLPackage/DX1_RLSetup/tasks/velocity/velocity_env_cfg.py b/DX1_RL/DX1_RLPackage/DX1_RLSetup/tasks/velocity/velocity_env_cfg.py
index f70afde..138415b 100644
--- a/DX1_RL/DX1_RLPackage/DX1_RLSetup/tasks/velocity/velocity_env_cfg.py
+++ b/DX1_RL/DX1_RLPackage/DX1_RLSetup/tasks/velocity/velocity_env_cfg.py
@@ -23,10 +23,9 @@ import isaaclab_tasks.manager_based.locomotion.velocity.mdp as mdp
 from isaaclab.terrains.config.rough import ROUGH_TERRAINS_CFG
 
 
+### Scene
 @configclass
 class MySceneCfg(InteractiveSceneCfg):
-    """Configuration for the terrain scene with a legged robot."""
-
     # ground terrain
     terrain = TerrainImporterCfg(
         prim_path="/World/ground",
@@ -47,8 +46,10 @@ class MySceneCfg(InteractiveSceneCfg):
         ),
         debug_vis=False,
     )
-    # robots
+
+    # DX1
     robot: ArticulationCfg = MISSING
+
     # sensors
     height_scanner = RayCasterCfg(
         prim_path="{ENV_REGEX_NS}/Robot/base",
@@ -59,6 +60,7 @@ class MySceneCfg(InteractiveSceneCfg):
         mesh_prim_paths=["/World/ground"],
     )
     contact_forces = ContactSensorCfg(prim_path="{ENV_REGEX_NS}/Robot/.*", history_length=3, track_air_time=True)
+
     # lights
     sky_light = AssetBaseCfg(
         prim_path="/World/skyLight",
@@ -69,11 +71,7 @@ class MySceneCfg(InteractiveSceneCfg):
     )
 
 
-##
-# MDP settings
-##
-
-
+### MDP settings
 @configclass
 class CommandsCfg:
     """Command specifications for the MDP."""
@@ -94,15 +92,11 @@ class CommandsCfg:
 
 @configclass
 class ActionsCfg:
-    """Action specifications for the MDP."""
-
     joint_pos = mdp.JointPositionActionCfg(asset_name="robot", joint_names=[".*"], scale=0.7, use_default_offset=True)
 
 
 @configclass
 class ObservationsCfg:
-    """Observation specifications for the MDP."""
-
     @configclass
     class PolicyCfg(ObsGroup):
         """Observations for policy group."""
@@ -269,7 +263,7 @@ class TerminationsCfg:
     # not sure what is optimal for this
     base_contact = DoneTerm(
         func=mdp.illegal_contact,
-        params={"sensor_cfg": SceneEntityCfg("contact_forces", body_names="base"), "threshold": 500.0},
+        params={"sensor_cfg": SceneEntityCfg("contact_forces", body_names="base"), "threshold": 1.0},
     )
 
 
@@ -280,11 +274,6 @@ class CurriculumCfg:
     terrain_levels = CurrTerm(func=mdp.terrain_levels_vel)
 
 
-##
-# Environment configuration
-##
-
-
 @configclass
 class LocomotionVelocityRoughEnvCfg(ManagerBasedRLEnvCfg):
     """Configuration for the locomotion velocity-tracking environment."""
@@ -302,15 +291,16 @@ class LocomotionVelocityRoughEnvCfg(ManagerBasedRLEnvCfg):
     curriculum: CurriculumCfg = CurriculumCfg()
 
     def __post_init__(self):
-        """Post initialization."""
         # general settings
         self.decimation = 4
         self.episode_length_s = 20.0
+
         # simulation settings
         self.sim.dt = 0.005
         self.sim.render_interval = self.decimation
         self.sim.physics_material = self.scene.terrain.physics_material
         self.sim.physx.gpu_max_rigid_patch_count = 10 * 2**15
+
         # update sensor update periods
         # we tick all the sensors based on the smallest update period (physics update period)
         if self.scene.height_scanner is not None:
diff --git a/DX1_RL/logs/rsl_rl/DX1_flat/2025-07-04_20-58-32/exported/policy.pt b/DX1_RL/logs/rsl_rl/DX1_flat/2025-07-04_20-58-32/exported/policy.pt
index 44c9464..e0df12f 100644
Binary files a/DX1_RL/logs/rsl_rl/DX1_flat/2025-07-04_20-58-32/exported/policy.pt and b/DX1_RL/logs/rsl_rl/DX1_flat/2025-07-04_20-58-32/exported/policy.pt differ